### ISSUE TITLE
Where are the tests for the package?

### ISSUE THREAD

**Issue by rabarar** [madmann8/learningGo#1]
I don't see the tests in the package? Did you write them??

**Comment by madmann8** [comment]
No, I completely forgot about those :). I'll add them and the GoDoc
information tomorrow and check back in.

Thank You,
Luke M

On Fri, Apr 29, 2016 at 10:32 PM, Rob Baruch notifications@github.com
wrote:

> I don't see the tests in the package? Did you write them??
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> https://github.com/madmann8/learningGo/issues/1

## 

From,
Luke Mann
